wallet/psbt_test.go

```diff
@@ -354,6 +392,27 @@ func assertTxInputs(t *testing.T, packet *psbt.Packet,
 	}
 }
 
+func assertChangeOutputScope(t *testing.T, pkScript []byte,
+	changeScope *waddrmgr.KeyScope) {
+
+	// We check if the pkScript has the right type.
```

Comment:
nit: this part of the comment could be the function's godoc comment. And the next line is overflowing the 80 char limit slightly.